### ISSUE TITLE
Trying to detect what is the users system preference

### ISSUE THREAD

**Issue by AdamBD** [nuxt-community/color-mode-module#117]
I am using the following line to set the users color mode preference:       
`this.$nuxt.$colorMode.preference = this.checked ? "dark" : "light";`

As I understand it this should set the default preference of the user. When requesting the preference of the user I am then getting
this.$nuxt.$colorMode.preference = 'system'

I assume that here system is representing that this module will check for the stored cookie on the users browser and take the literal string value from there.

When trying to check what that value is I am still getting system (However when printing the entire ColorMode object I am actually getting 'dark'.

What is the correct way to understand what is the VALUE of the system preference?

![image](https://user-images.githubusercontent.com/1134525/148643663-a11184ae-5d14-47f5-a013-e2aa24833717.png)

**Comment by nklsw** [comment]
I'm encountering the same issue (using v3.0.1). Any updates?

**Comment by Ovab** [comment]
Yes, preference is just what the user set as what they prefer (handy for a mode switcher).

colormode value = the device's preference if colormode preference = system
